packages/message-predicates/Forc.lock

```diff
@@ -13,9 +13,9 @@ dependencies = [
 
 [[package]]
 name = "core"
-source = "path+from-root-AD80769CAE44474A"
+source = "path+from-root-AC247AEA3D39B916"
 
 [[package]]
 name = "std"
-source = "git+https://github.com/fuellabs/sway?tag=v0.46.0#e75f14b03636bc96751a6760304a1a6d3eb5937d"
+source = "git+https://github.com/fuellabs/sway?tag=v0.47.0#34265301c6037d51444899a99df1cfc563df6016"
```

Comment:
Weren't we using 0.48.1 above? Why is this updated to 0.47.0?

Reply:
Not really sure about what happened with this one. Apparently forc does not update the deps automatically from the workspace. I 'll raise it just in case too

Reply:
Fixed